### ISSUE TITLE
Ability to define WHERE step definitions / features are located

### ISSUE THREAD

**Issue by jthebison** [cucumber/godog#506]
### 🤔 What's the problem you're trying to solve?
Attempt to centralize test code in one area, instead of all of the test code scattered across the entire project. I understand it's not necessarily idiomatic Go code, but it's more in line with how BDD test code is written in other languages.

I'm frustrated when I'm trying to run a regression on my entire project, and I have to find individual files instead of being able to run  and find all of my test code within one folder
### ✨ What's your proposed solution?
Some way to specify where the source of the step definitions and feature files should be. 

This could be either a cli argument, or some sort of setting file, or even a field that can be set in the context struct.
### ⛏ Have you considered any alternatives or workarounds?
When it comes to location of the files it seems diifficult to see any workaround that aren't being able to move the files.

### 📚 Any additional context?

For example, within the Java Cucumber support, you can define specifically where the features are located. The feature files themselves are able to map to step definitions anywhere in the project.

![image](https://user-images.githubusercontent.com/60556727/196227111-86b58eb3-0e76-4745-b3fa-fec3c91e3762.png)
----

*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/ISSUE_TEMPLATE/feature_request.md)*

**Comment by gnoe** [comment]
Hello @sumusjack, you can define where features/steps definitions are located. Take this example as reference https://github.com/cucumber/godog/blob/main/_examples/assert-godogs/godogs_test.go.
In [contexts initialization](https://github.com/cucumber/godog/blob/main/_examples/godogs/godogs_test.go#L39) you can define the function where all scenario contexts are defined.
And in [options](https://github.com/cucumber/godog/blob/main/_examples/godogs/godogs_test.go#L25) you can set the [Paths](https://github.com/cucumber/godog/blob/main/internal/flags/options.go#L57) property with your features' paths.

By doing so you can centralize your tests in one place and when you run `go test ` in your project the BDD tests will run as well

**Comment by gabrielcartellivia** [comment]
Hello @gnoe. Can I define where the tests files are located? 

E.g.:
features
|__ godog.feature
services
|__ godog_test.go

**Comment by roskee** [comment]
Yes. As @gnoe has said, you can specify the path of your feature file `godog.feature` within your test `godog_test.go` using the `Paths` field.

**Comment by gabrielcartellivia** [comment]
![image](https://user-images.githubusercontent.com/72199278/214654561-b1a939b3-552e-47a5-bb6f-8a2555387712.png)

This isn't working. What am i doing wrong?

**Comment by roskee** [comment]
add the file names to the paths list.
```
...
Paths: []string{"..features/users.feature"}
...
```

**Comment by gabrielcartellivia** [comment]
> add the file names to the paths list.
> 
> ```
> ...
> Paths: []string{"../features/users.feature"}
> ...
> ```

![image](https://user-images.githubusercontent.com/72199278/214656095-f7328f78-c29c-4661-829b-53f64e9951dc.png)

didn't work

**Comment by roskee** [comment]
What is the command you executed?

**Comment by gabrielcartellivia** [comment]
Just godog run

**Comment by roskee** [comment]
run `go test` inside the `services` directory.

**Comment by gabrielcartellivia** [comment]
worked! thanks @roskee

**Comment by gnoe** [comment]
Thank you @roskee for replying and sorry @gabrielcartellivia for didn’t reply on time.
I'm glad that it worked!

**Comment by mirogta** [comment]
@gabrielcartellivia can we close this issue?
